README.md

```diff
@@ -542,22 +543,40 @@ Note that the data is still a placeholder and hardcoded in the Next.js code. We
 To do so, we can change the `posts` definition with the following:
 
 ```tsx
+interface BlogIndexPage {
+  title: string;
+  intro: string;
+}
+
 interface BlogPage {
   id: number;
   meta: {
-    type: string;
     slug: string;
-    first_published_at: string;
   };
   title: string;
+  date: string;
   intro: string;
 }
 
 export default async function BlogIndex() {
+  const indexPages = await fetch(
+    `http://localhost:8000/api/v2/pages/?${new URLSearchParams({
+      type: "blog.BlogIndexPage",
+      fields: "intro",
+    })}`,
+    {
+      headers: {
+        Accept: "application/json",
+      },
+    }
+  ).then((response) => response.json());
```

Comment:
If we do feel ok about the two separate requests, then I think this needs addressing in the tutorial’s contents to explain why this is ok.

Reply:
How else would people get requests from multiple functions into a single page without using GraphQL to condense them instead? Most real-world scenarios are going to have different types of data on the same page.

We can definitely add some more context afterwards.